README.md

```diff
@@ -1,11 +1,13 @@
-<h> A/B Testing/h>
+# A/B Testing
 
 <ins>**Overview**</ins>
+
 Also known as split testing, is a method of comparing two versions of a webpage or a feature of a user interface
```

Comment:
"it is a method.."
You don't need to document what A/B testing is in the readme, it would be better if you just find a nice article online and link it (see markdown for using links).
Use the readme only to describe the current project.